lib/geokit/geocoders/bing.rb

```diff
@@ -55,7 +55,7 @@ def self.extract_location(xml)
         zip:            'Address/PostalCode',
         country:        'Address/CountryRegion',
         lat:            'Point/Latitude',
-        lng:            'Point/Longitude'
+        lng:            'Point/Longitude',
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.